controls/busyindicator/integration.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Integration
+page_title: .NET MAUI BusyIndicator Documentation - CollectionView Integration
+description: Learn more about the option to integrate BusyIndacator with CollectionView control.
+position: 2
+previous_url: /controls/busyindicator/integrate-with-listview
+slug: busyindicator-integration
+---
+
+# .NET MAUI BusyIndicator Integration with CollectionView
+
+The Telerik BusyIndicator for .NET MAUI is useful when you want to display a notification to the end-users of the application while a long-running operation, such as loading data from a service, is currently in progress.
```

Comment:
Only use `end-user` as an adjective or adverb. Here, you need to go with `end users`.

```suggestion
The Telerik BusyIndicator for .NET MAUI is useful when you want to display a notification to the application's end users during a long-running operation. For example, you might need to display a busy indicator while the application is loading data from a service.
```